pyprof/nvtx/nvmarker.py

```diff
@@ -42,13 +42,28 @@
 import json
 from .config import Config
 
-# Flag to indicate if wrapper_func() should inject nvtx or 
+# Global state variables
+call_id = 0
+capture_input_ops = 0
+patch_list = []  ## Keep track of nested calls to wrapper
+op_to_out_tensor_map = {}
```

Comment:
Can we not use globals. A singleton class might be?

Reply:
should be in place now